Library/Formula/mitie.rb

```diff
@@ -14,27 +14,34 @@ class Mitie < Formula
 
   depends_on :python if MacOS.version <= :snow_leopard
 
+  option "with-models", "Install v0.2 models (~415MB)"
```

Comment:
And it installs a C library too, so there are definitely use-cases that don't require the models.